ocs/site_config.py

```diff
@@ -229,6 +229,12 @@ def from_dict(cls, data, parent=None):
             the Agent instance, as a way of finding the right
             InstanceConfig.
 
+        ``agent-exe`` (str, optional)
+            The Agent executable.  This
+            may be matched against the agent_exe name provided by
+            the Agent instance, as a way of finding the right
+            InstanceConfig.
+
```

Comment:
I don't think I had any particular use in mind; I was just trying to mimic the class-matching feature in case it was somehow important. On that basis, I don't think I would object to removing it, now or later.